src/index.ts

```diff
@@ -26,6 +26,7 @@ const availableOptions: string[] = [
 const option: string = process.argv[2];
 
 const main = async () => {
+  const helpFlag = Boolean(process.argv.filter((arg) => arg === "--help")[0]);
```

Comment:
Should we also accept `-h`?
```suggestion
  const helpFlag = Boolean(process.argv.filter((arg) => ["--help", "-h"].includes(arg))[0]);
```